app/models/skill.rb

```diff
@@ -0,0 +1,6 @@
+class Skill < ActiveRecord::Base
+  SKILL_LEVELS = {"Beginner": 1, "Intermediate": 3, "Expert": 5, "Leader": 7}
+  validates_presence_of :name
+  validates_presence_of :level
```

Comment:
@jimmzhu i'm not entirely sure i can access it in the controller like I do currently here:

https://github.com/dhakiki/nucleus-directory/pull/1/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R103

(doing something similar right below it too:)

https://github.com/dhakiki/nucleus-directory/pull/1/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R107